pkg/sql/opt/exec/execbuilder/testdata/delete

```diff
@@ -221,7 +221,7 @@ vectorized: true
 
 # Tracing tests for fast delete.
 statement ok
-CREATE TABLE a (a INT PRIMARY KEY)
+CREATE TABLE a (a DECIMAL PRIMARY KEY)
 
 # Delete range operates in chunks of 600 (defined by row.TableTruncateChunkSize).
```

Comment:
nit: drive-by that we should do `s/TableTruncateChunkSize/deleteRangeChunkSize/`.